test/mock_data.py

```diff
@@ -170,5 +170,6 @@
 
 IMAGE_GET_DATA_ALL_VALUES = {
     **IMAGE_GET_METADATA_DATA_ALL_VALUES,
-    "url": ANY,
+    "inline_url": ANY,
+    "download_url": ANY,
```

Comment:
I think it should probably either be
```suggestion
    "url": ANY,
    "download_url": ANY,
```
(Maybe "view_url")
Or otherwise
```suggestion
    "inline_url": ANY,
    "attachment_url": ANY,
```
I personally prefer the former, probably with `view_url`?

Reply:
I agree I think download_url with view_url is better